oci/table_oci_core_boot_volume_backup.go

```diff
@@ -0,0 +1,291 @@
+	return &plugin.Table{
+		Name:        "oci_core_boot_volume_backup",
+		Description: "OCI Core Boot Volume Backup",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AnyColumn([]string{"id"}),
+			Hydrate:    getBootVolumeBackup,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listBootVolumeBackups,
+		},
+		GetMatrixItem: BuildCompartementRegionList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The OCID of the boot volume backup..",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "display_name",
+				Description: "A user-friendly name for the boot volume backup.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "boot_volume_id",
+				Description: "The OCID of the boot volume.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromCamel(),
+			},
+			{
+				Name:        "expiration_time",
+				Description: "The date and time the volume backup will expire and be automatically deleted.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+			{
+				Name:        "image_id",
+				Description: "The image OCID used to create the boot volume the backup is taken from.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "kms_key_id",
+				Description: "The OCID of the Key Management master encryption assigned to the boot volume backup.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "The date and time the boot volume backup was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+			{
+				Name:        "time_request_received",
+				Description: "The date and time the request to create the boot volume backup was received.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+			{
+				Name:        "size_in_gbs",
+				Description: "The size of the boot volume, in GBs.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("SizeInGBs"),
+			},
+			{
+				Name:        "source_boot_volume_backup_id",
+				Description: "The OCID of the source boot volume backup.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "source_type",
+				Description: "Specifies whether the backup was created manually, or via scheduled backup policy.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "unique_size_in_gbs",
+				Description: "The size used by the backup, in GBs.",
+				Type:        proto.ColumnType_INT,
+				Transform:   transform.FromField("UniqueSizeInGBs"),
+			},
+
+			// JSON columns
+			{
+				Name:        "lifecycle_state",
+				Description: "The current state of a boot volume backup.",
+				Type:        proto.ColumnType_JSON,
```

Comment:
```suggestion
				Type:        proto.ColumnType_STRING,
```
And this should be the 4th column.